src/extractor/guidance/roundabout_handler.cpp

```diff
@@ -270,13 +270,30 @@ RoundaboutType RoundaboutHandler::getRoundaboutType(const NodeID nid) const
     std::set<NodeID> roundabout_nodes; // needs to be sorted
 
     // this value is a hard abort to deal with potential self-loops
+    const auto countRoundaboutFlags = [&](const NodeID at_node) {
+        // this would be nicer as boost::count_if, but our integer range does not support these
+        // range based handlers
```

Comment:
Could you ticket the range problem? You have a better understanding of what we need to get our ranges working correctly.